README.md

```diff
@@ -3,7 +3,7 @@
 localectl
 ===========
 
-Basic description for localectl
+This role will set systmctl; System Locale, VC Keymap, and X11 Layout settings
```

Comment:
This role does nothing with systemctl (or systmctl), as far as I can tell. Description should probably match terminology that localectl itself uses. This is based on the localectl manpage: "Control the system locale and keyboard layout settings using localectl".

This also applies to the description in meta/main.yml.